### DISCUSSION TITLE
For React Native which one should I use between nativewind or tailwindcss-react-native ?

### DISCUSSION THREAD

**Discussion by robyhuzwandar** [nativewind/nativewind#112] [Q&A] [answered]
seems nativewind documentation still uses tailwindcss-react-native. so which one should I use?

<img width="933" alt="image" src="https://user-images.githubusercontent.com/20414047/178467525-b4ed8f15-61a3-4012-9acb-a0b864fff8d1.png">

**Comment by marklawlor** [comment, marked as answer]
I just pushed an update ~5mins ago which clarifies this :stuck_out_tongue_winking_eye:  https://www.nativewind.dev/quick-starts/react-native-cli.

Sorry the docs were slightly outdated

**Reply by robyhuzwandar** [reply]
superb! 🎉⚡️

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
